media/js/base/sticky-promo.js

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var closeButton = document.querySelector('.mzp-c-sticky-promo-close');
+
+    closeButton.addEventListener('click', function(e) {
+        e.preventDefault();
+        e.currentTarget.parentNode.classList.add('mzp-a-fade-out');
+    }, false);
+
+})();
```

Comment:
This code is available to you via `protocol/js/protocol-sticky-promo.js` so we can add that to the JS bundles instead :D